internal/core/app_deploy_destroy.go

```diff
@@ -114,11 +118,30 @@ func (a *App) destroyDeploy(
 	}
 	defer c.Close()
 
-	_, _, err = a.doOperation(ctx, a.logger.Named("deploy"), &deployDestroyOperation{
+	_, destroyment, err := a.doOperation(ctx, a.logger.Named("deploy"), &deployDestroyOperation{
 		Component:  c,
 		Deployment: d,
 	})
-	return err
+
+	destroyProto, ok := destroyment.(*pb.Deployment)
+	if !ok {
+		return err
+	}
+
+	if len(destroyProto.DeclaredResources) > 0 {
+		a.UI.Output("These resources were not destroyed for app %q:", a.ref.Application, terminal.WithWarningStyle())
+		for _, resource := range destroyProto.DeclaredResources {
+			a.UI.Output("- "+resource.Name, terminal.WithWarningStyle())
+		}
+		if len(destroyProto.DestroyedResources) > 0 {
+			a.UI.Output("These resources were destroyed for app %q:", a.ref.Application, terminal.WithSuccessStyle())
+			for _, resource := range destroyProto.DestroyedResources {
+				a.UI.Output("- "+resource.Name, terminal.WithSuccessStyle())
+			}
+		}
+	}
```

Comment:
It might be nice to build up these strings in the loops and do a single `a.UI.Output` once rather than for each entry